File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Blazor.md

```diff
@@ -0,0 +1,342 @@
+            using (PdfDocument pdfDocument = render.ConvertToPDF(document))
+            {
+                //Save the PDF document to MemoryStream.
+                using (MemoryStream outputStream = new MemoryStream())
+                {
+                    pdfDocument.Save(outputStream);
+                    outputStream.Position = 0;
+                    //Download PDF file in the browser.
+                    await JS.SaveAs("Output.pdf", outputStream.ToArray());
+                }
+            }
+        }
+    }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 7: Create a class file with FileUtils name and add the following code to invoke the JavaScript action to download the file in the browser.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+public static class FileUtils
+{
+    public static ValueTask<object> SaveAs(this IJSRuntime js, string filename, byte[] data)
+       => js.InvokeAsync<object>(
+            "saveAsFile",
+            filename,
+            Convert.ToBase64String(data));
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 8: Add the following JavaScript function in the Index.html file present under ``wwwroot``.
+
+{% tabs %}
+
+{% highlight HTML %}
+
+<script type="text/javascript">
+    function saveAsFile(filename, bytesBase64) {
+        if (navigator.msSaveBlob) {
+            //Download document in Edge browser
+            var data = window.atob(bytesBase64);
+            var bytes = new Uint8Array(data.length);
+            for (var i = 0; i < data.length; i++) {
+                bytes[i] = data.charCodeAt(i);
+            }
+            var blob = new Blob([bytes.buffer], { type: "application/octet-stream" });
+            navigator.msSaveBlob(blob, filename);
+        }
+        else {
+            var link = document.createElement('a');
+            link.download = filename;
+            link.href = "data:application/octet-stream;base64," + bytesBase64;
+            document.body.appendChild(link); // Needed for Firefox
+            link.click();
+            document.body.removeChild(link);
+        }
+    }
+</script>
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 9: Add the following code snippet in the razor file of Navigation menu in the Shared folder.
+
+{% tabs %}
+
+{% highlight HTML %}
+
+<li class="nav-item px-3">
+    <NavLink class="nav-link" href="docio">
+        <span class="oi oi-list-rich" aria-hidden="true"></span> Convert Word to PDF
+    </NavLink>
+</li>
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Getting-Started/Blazor/Client-side-application).
+
+By executing the program, you will get the **PDF document** as follows.
+
+![Blazor WASM app output Word document](WordToPDF_images/OutputImage.png)
+
+N> Even though Word library works in WASM app, it is recommended to use server deployment. Since the WASM app deployment increases the application payload size.
```

Comment:
N> To convert Word to PDF, it is necessary to access the font stream internally. However, this cannot be done automatically in a Blazor WASM application. Therefore, we recommend using a Server app, even though Word to PDF conversion works in a WASM app.